Backend/app/rider-platform/rider-app/Main/src/Domain/Types/Booking/API.hs

```diff
@@ -254,7 +258,9 @@ buildBookingAPIEntity booking personId = do
   mbActiveRide <- runInReplica $ QRide.findActiveByRBId booking.id
   mbRide <- runInReplica $ QRide.findByRBId booking.id
   -- nightIssue <- runInReplica $ QIssue.findNightIssueByBookingId booking.id
-  fareBreakups <- bool (pure []) (runInReplica $ QFareBreakup.findAllByBookingId booking.id) (booking.status == COMPLETED)
+  -- fareBreakups <- bool (pure []) (runInReplica $ QFareBreakup.findAllByBookingId booking.id) (booking.status == COMPLETED)
+  fareBreakups <- maybe (pure []) (\rideId -> QFareBreakup.findAllByEntityIdAndEntityType rideId.getId DFareBreakup.RIDE) (mbRide <&> (.id))
```

Comment:
@dushyant-2002 For backward compatibility, If any ride which is created before the release that will have Fare Breakup stored with entity_type as BOOKING and booking_id. Also add runInReplica, to avoid this query from going to Master DB.
So, we can write something like this below:
```
fareBreakups <- ((\rideId -> runInReplica  QFareBreakup.findAllByEntityIdAndEntityType rideId.getId DFareBreakup.RIDE) <$> (mbRide <&> (.id)) <|> (runInReplica $ QFareBreakup.findAllByBookingId booking.id)) >>= fromMaybeM (pure [])
```